suite-native/module-home/src/components/EmptyHomeScreenPlaceholder.tsx

```diff
@@ -0,0 +1,44 @@
+import { useNavigation } from '@react-navigation/native';
+
+import { Button, Card, Pictogram, VStack } from '@suite-native/atoms';
+import { useTranslate } from '@suite-native/intl';
+import {
+    RootStackParamList,
+    RootStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const cardStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginBottom: utils.spacings.extraLarge,
+    padding: utils.spacings.extraLarge,
+}));
+
+export const EmptyHomeScreenPlaceholder = () => {
```

Comment:
prefer HomescreenEmptyState

Reply:
Is this even used? I think we are using loading state of Graph now instead ?

Reply:
If the discovery finishes without any account discovered, it defaults to this new empty screen placeholder.

Reply:
<img width="232" alt="Screenshot 2023-10-05 at 21 10 48" src="https://github.com/trezor/trezor-suite/assets/26143964/91f1ebe0-4ddf-4517-8923-d0ca7513fe3b">

Reply:
I see thanks. About the name -> I don't care I like both of them :D